osquery/CMakeLists.txt

```diff
@@ -304,6 +304,10 @@ if(NOT DEFINED ENV{SKIP_TESTS})
   add_executable(osquery_tests main/tests.cpp ${OSQUERY_TESTS})
   ADD_DEFAULT_LINKS(osquery_tests FALSE)
   target_link_libraries(osquery_tests gtest gmock libosquery_testing)
+  if(WINDOWS)
+    # Linked for the WMI libs
+    target_link_libraries(osquery_tests Release/osquery_additional)
```

Comment:
This isn't a good workaround. To make this work with the build the WMI classes will need to be available to core. Mind opening an issue for this need to refactor?